packages/test/src/api/string-response.ts

```diff
@@ -17,21 +17,26 @@ import * as Common from './common'
 import * as CLI from './cli'
 import * as ReplExpect from './repl-expect'
 
-interface TestParam {
+interface Param {
   command: string
   expect: string
   exact: boolean
 }
 
 export class TestStringResponse {
-  public constructor(test: TestParam) {
+  // eslint-disable-next-line no-useless-constructor
+  public constructor(public readonly param: Param) {}
+
+  public string() {
+    const { command, expect, exact } = this.param
+
     describe(`string response ${process.env.MOCHA_RUN_TARGET || ''}`, function(this: Common.ISuite) {
       before(Common.before(this))
       after(Common.after(this))
 
       it('should return a string', () =>
-        CLI.command(test.command, this.app)
-          .then(ReplExpect.okWithString(test.expect, test.exact))
+        CLI.command(command, this.app)
+          .then(ReplExpect.okWithString(expect, exact))
           .catch(Common.oops(this, true)))
     })
   }
```

Comment:
Modified the string response test file to avoid using `new` without storing the result. See Eslint Rule no-new: https://eslint.org/docs/1.0.0/rules/no-new